src/test/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalkTest.java

```diff
@@ -149,7 +150,7 @@ public void ddm2Emd() throws Exception {
 
     private String emdElementFrom(String ddm) throws CrosswalkException, XMLSerializationException {
         EasyMetadata emd = new Ddm2EmdCrosswalk(null).createFrom(ddm);
-        return new EmdMarshaller(emd).getXmlString();
+        return new EmdMarshaller(emd).getXmlString().replaceAll("@@@SURROGATE-PAIR:(.*?)@@@", "&#$1;");
```

Comment:
You only did this replace here in the test code. Does that mean the `"@@@SURROGATE-PAIRxxx@@@"` is kept in the 'normal source code'?